### DISCUSSION TITLE
can't it work with esphome?

### DISCUSSION THREAD

**Discussion by zetazetterberg** [ScratMan/HASmartThermostat#232] [Q&A]
can't get it to work with esphome. the files are placed in custom components. with when I go to upload the code to the esp device, it says that smart_thermostat does not exist.
of course the files are in HA config where esphome is also located.

I think I have the smart thermostat files in the wrong place?

**Comment by Chupaka** [comment]
ESPHome?.. This is purely HA component, you can use it to control ESPHome devices, but it runs on HA, not ESPHome device
